logging/kibana.ts

```diff
@@ -0,0 +1,29 @@
+import fetch, { RequestInit } from "node-fetch";
+
+class KibanaClient {
```

Comment:
I'm surprised there isn't a Kibana client published by Elastic.

Reply:
So was I - I'm not sure if it's about stability, or just another library to maintain...